### PR TITLE
FIX: Don't create a reviewable when sending PM to user in chat

### PR DESCRIPTION
**Pull request by jancernik**

### CODE REVIEW

File: plugins/chat/spec/lib/chat/review_queue_spec.rb [hunk 1]
```diff
@@ -184,6 +184,13 @@
         expect(pm_topic.title).to eq("Your chat message in \"#{chat_channel.title(message.user)}\"")
       end
 
+      it "doesn't create a revieable" do
```
Comment (pmusaraj) [inline plugins/chat/spec/lib/chat/review_queue_spec.rb:187]:
```suggestion
      it "doesn't create a reviewable" do
```

**Review by pmusaraj — COMMENTED** [review]
I'm not quite following this use case, can you describe a bit under which conditions we want to skip creating a reviewable?

**Comment by jancernik** [comment]
The change is mainly to improve the consistency between flagging a chat message and flagging a post. This, in particular skips creating a reviewable when sending a PM to the user using the option "Send @user a message" from the flag menu.

Meta topic: https://meta.discourse.org/t/send-message-to-user-flags-create-review-item-in-chat/272362

**Review by pmusaraj — APPROVED** [review]
Thanks for the explanation, 👍 !

**Review by jjaffeux — APPROVED** [review]

**Comment by jjaffeux** [comment]
@pmusaraj @jancernik Thanks, I just merged it :+1:

**Pull request merged**